src/server.tsx

```diff
@@ -59,7 +59,7 @@ interface Epic {
     css: string;
 }
 
-const [, fetchDefaultEpicContentCached] = memoiseAsync(fetchDefaultEpicContent);
+const [, fetchDefaultEpicContentCached] = cacheAsync(fetchDefaultEpicContent, 60 * 5);
```

Comment:
Thoughts on extracting `60 * 5` out to a variable, just to make is super clear? `const fiveMinutes = 60 * 5;`?